File: src/class9/collections/exercise3/java/Prova.java
package class9.collections.exercise3.java;

import java.util.Set;

public class Prova {

    public void somaTotal(Set<Integer> conjuntoDeInteiros) {
        int soma = 0;
        for (int item : conjuntoDeInteiros) {
            soma += item;
        }
        System.out.println("Valor total da soma = " + soma);
    }
}

File: app/src/main/java/intellij/class9/collections/exercise3/kotlin/Prova.kt
package intellij.class9.collections.exercise3.kotlin

class Prova {
    fun somaTotal(conjuntoDeInteiros: Set<Int>) {
        var soma = 0
        for (item in conjuntoDeInteiros) {
            soma += item
        }
        println("Valor total da soma = $soma")
    }
}